osu.Game.Rulesets.Osu/Mods/OsuModRandom.cs

```diff
@@ -287,7 +333,25 @@ private Vector2 clampToPlayfieldWithPadding(Vector2 position, float padding)
 
         private class RandomObjectInfo
         {
-            public float AngleRad { get; set; }
+            /// <summary>
+            /// The jump angle from the previous hit object to this one, relative to the previous hit object's jump angle.
+            /// </summary>
+            /// <remarks>
+            /// <see cref="RelativeAngle"/> of the first hit object in a beatmap represents the absolute angle from playfield center to the object.
+            /// </remarks>
+            /// <example>
+            /// If <see cref="RelativeAngle"/> is 0, the player's cursor doesn't need to change its direction of movement when passing
+            /// the previous object to reach this one.
+            /// </example>
+            public float RelativeAngle { get; set; }
+
+            /// <summary>
+            /// The jump distance from the previous hit object to this one.
+            /// </summary>
+            /// <remarks>
+            /// <see cref="Distance"/> of the first hit object in a beatmap is relative to the playfield center.
+            /// </remarks>
+            public float Distance { get; set; }
```

Comment:
Maybe should be called `DistanceFromPrevious`? Not sure.

Reply:
Yeah, I think `DistanceFromPrevious` is better.